src/map/battle.cpp

```diff
@@ -593,6 +593,13 @@ int64 battle_attr_fix(struct block_list *src, struct block_list *target, int64 d
 				}
 				break;
 		}
+
+		if (tsc->data[SC_MAGIC_POISON])
```

Comment:
Magic poison  is effect to magic attack only.

Reply:
The description from Sigma says "Target under this effect will take more damage from all property by 50%.". To me, this sounds like weapon, misc, and magic. Or am I interpreting this wrong?

Reply:
You are correct confirm by this pic.

![image](https://user-images.githubusercontent.com/35770095/89714430-d9dc4700-d9c8-11ea-852e-9692161a6b28.png)